### PR TITLE
Add Blog, YouTube, and Laser URLs

### PR DESCRIPTION
**Pull request by vandahm**
This PR adds four URLs to the list.

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -1,9 +1,13 @@
 <?php
 $links = array(
+    'blog' => 'https://tcmaker.org/news',
```
Comment (vandahm) [inline index.php:3]: @automatingerror You can comment on specific lines of code.

Comment (automatingerror) [reply index.php:3]: Is the Facebook Group the only Facebook resource we will be linking with the shortener? If it isn't we may want to make the link more descriptive.

File: index.php [hunk 2]
```diff
@@ -1,9 +1,13 @@
...
     'cal' => 'https://tcmaker.org/calendar',
     'classes' => 'https://tcmaker.org/classes',
     'eb' => 'https://www.eventbrite.com/o/twin-cities-maker-440413464',
     'fb' => 'https://www.facebook.com/groups/TwinCitiesMaker/',
```
Comment (automatingerror) [inline index.php:7]: Consider changing line to:
`    'fbgrp' => 'https://www.facebook.com/groups/TwinCitiesMaker/',`
if we will be linking to other facebook resources.

File: index.php (hunk 1 above)
Comment (vandahm) [reply index.php:3]: For now, it is. I think `fb` should point to our main Facebook presence, whether that's the group or the page. I don't care which one it is, so long as we pick one and stick with it.

**Review by automatingerror — COMMENTED** [review]
Consider if we will be linking to other facebook resources.

File: index.php (hunk 2 above)
Comment (vandahm) [reply index.php:7]: Are there, in the handbook, other links to Facebook resources? If there aren't, then this issue is moot.

**Pull request merged**